src/main/java/com/streamr/client/StreamrClient.java

```diff
@@ -245,6 +249,10 @@ public void disconnect() throws ConnectionTimeoutException {
         }
     }
 
+    public void setErrorMessageHandler(ErrorMessageHandler errorMessageHandler) {
+        this.errorMessageHandler = errorMessageHandler;
+    }
+
```

Comment:
For consistency with the way the other parameters are passed, could be passed in `StreamrClientOptions` instead?

Reply:
I agree. However this should probably be done in separate PR.